public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta http-equiv="Content-Security-Policy" content="script-src 'self';">
+    <meta http-equiv="Content-Security-Policy" content="script-src 'unsafe-inline' 'self';">
```

Comment:
Hi @laemtl, this change is needed to allow developers to save changes and continue testing the user interface without having to re-build.